images/logging-eventrouter.yml

```diff
@@ -1,15 +1,28 @@
+content:
+  source:
+    dockerfile: Dockerfile
+    git:
+      branch:
+        # This repo does not follow the release- branch convention. Remove fallback when it does.
+        fallback: master
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/eventrouter.git
+distgit:
+  namespace: containers
```

Comment:
Not sure the `distgit` section is required here; containers must already be the default for images as most don't have this.